HLTrigger/Configuration/python/customizeHLTforMC.py

```diff
@@ -18,6 +18,13 @@ def customizeHLTforMC(process,_fastSim=False):
     process.hltParticleFlowRecHitHCAL.ApplyPulseDPG      = cms.bool(False)
     process.hltParticleFlowRecHitHCAL.LongShortFibre_Cut = cms.double(1000000000.0)
 
+  # customise hltHbhereco to use the Method 3 parametrisation for Monte Carlo (see #11091)
+  if 'hltHbhereco' in process.__dict__:
```

Comment:
Hi Martin,
in principle I fully agree with you, and I would push for cleaning up the
data-mc differences from the python configuration as soon as we can (CMSSW
7.6.0 final, or even 7.7/8.0 ?).

However, we need this to be working by today for 76x, 75x, and possibly
74x, so we have no hope of adding these parameters via the global tag, in
time.

Hence, I agreed with HCAL that we would support this change via a python
customization for the time being, and even to provide it for them...

...you do get to blame me for any mess this causes in the HLT, if it helps.

Salavat, the fillDescriptions method is not something used only by the HLT,
but a general CMSSW mechanism for catching errors in the modules'
configurations and validating their parameters.
While we should use it more extensively, I do not want to push on such
urgent time scale for changes that would make the code more difficult to
maintain, not easier.

So, let's do away with it for 76x/75x, and look for a better approach
afterwards, OK ?